setup-env/src/actionInput/index.js

```diff
@@ -42,12 +50,14 @@ class ActionInput {
     this.username = InputValidator.updateUsername(this.username);
     this.buildName = InputValidator.validateBuildName(this.buildName);
     this.projectName = InputValidator.validateProjectName(this.projectName);
+    this.githubToken = InputValidator.validateGithubToken(this.githubToken);
+    this.githubApp = InputValidator.validateGithubAppName(this.githubApp);
   }
 
   /**
    * Sets env variables to be used in the test script for BrowserStack
    */
-  setEnvVariables() {
+  async setEnvVariables() {
```

Comment:
Check backward compatibility

Reply:
The calling methods are already async, added await for these method calls. The functionality seems to be working fine anyways.